Gifski/AppDelegate.swift

```diff
@@ -125,14 +125,8 @@ extension NSColor {
 @NSApplicationMain
 final class AppDelegate: NSObject, NSApplicationDelegate {
 	@IBOutlet private weak var window: NSWindow!
-	var gifski: Gifski? {
-		didSet {
-			gifski?.onProgress = { [weak self] progress in
-				self?.updateProgress(progress)
-			}
-		}
-	}
-	var progress: Progress?
+
+	private var progressObserver: NSKeyValueObservation?
```

Comment:
Do we really need to keep a reference to it here? I tried having it as just a local variable in `startConversion` and it worked fine.

Reply:
I just tried it and the main window progress did not animate. Did you try on this branch, or could it be that progress was still an ivar when you tested?

Reply:
I tried again and you're right. No idea why it worked earlier...